nilearn/glm/contrasts.py

```diff
@@ -437,6 +459,26 @@ def _compute_fixed_effects_params(contrasts, variances, precision_weighted):
     else:
         fixed_fx_variance = np.mean(variances, 0) / len(variances)
         fixed_fx_contrasts = np.mean(contrasts, 0)
+    dim = 1
+    contrast_type = 't'
+    fixed_fx_contrasts_ = fixed_fx_contrasts
+    if len(fixed_fx_contrasts.shape) == 2:
+        dim = fixed_fx_contrasts.shape[0]
+        if dim > 1:
+            contrast_type = 'F'
+    else:
+        fixed_fx_contrasts_ = fixed_fx_contrasts[np.newaxis]
+    con = Contrast(effect=fixed_fx_contrasts_, variance=fixed_fx_variance,
+                   dim=dim, dof=np.sum(dofs), contrast_type=contrast_type)
+    fixed_fx_z_score = con.z_score()
+    fixed_fx_stat = con.stat_
 
-    fixed_fx_stat = fixed_fx_contrasts / np.sqrt(fixed_fx_variance)
-    return fixed_fx_contrasts, fixed_fx_variance, fixed_fx_stat
+    """
+    if contrast_type == 't':
+        if len(fixed_fx_z_score.shape) == 2:
+            fixed_fx_z_score = fixed_fx_z_score[:, 0]
+        if len(fixed_fx_stat.shape) == 2:
+            fixed_fx_stat = fixed_fx_stat[:, 0]
+    """
```

Comment:
why is there a doc string? 
should this be removed?

Reply:
Yes L476-482 should be removed. It looks like unresolved merge conflict

Reply:
done